doc/sphinx/source/community/release_strategy.rst

```diff
@@ -248,6 +248,7 @@ These are the detailed steps to take to make a release.
 
    - A release branch is created and branch protection rules are set up so only the release manager (i.e. the person in charge of the release branch) can push commits to that branch.
    - Make a release candidate with the release branch following the :ref:`ESMValCore release instructions <esmvalcore:how-to-make-a-release>`.
+   - Activate the release candidate channel in the ``environment.yml`` of ESMValTool and adjust the pin on ESMValCore after each release candidate (e.g. ``esmvalcore==2.8.0rc1``).
```

Comment:
"Activate" sounds a bit too much like `conda activate` - maybe rename to "uncomment item in environment file that adds the release candidate label channel to the list of conda channels we use"? A lot more verbose but a bit more explanation there, plus no "activate"

Reply:
Thanks V! Done in https://github.com/ESMValGroup/ESMValTool/pull/3096/commits/158591aab3ff7389c9dd27c3573f7d4b63f4f891. I guess I took inspiration from what's written in our [env file](https://github.com/ESMValGroup/ESMValTool/blob/main/environment.yml). But I don't have a strong opinion on the use of this word 😅